test/system/src/main/java/io/pravega/test/system/framework/services/PravegaSegmentStoreService.java

```diff
@@ -95,7 +94,6 @@ public void stop() {
     private App createPravegaSegmentStoreApp() {
         App app = new App();
         app.setId(this.id);
-        app.setBackoffSeconds(BACK_OFF_SECS);
```

Comment:
If we remove this line, then back is zero or does it have a default?

Reply:
The default values are : 
``` "backoffSeconds": 1,
    "backoffFactor": 1.15,
    "maxLaunchDelaySeconds": 3600,
```
controller is using these default values .

Reply:
The configuration was initially taken from PravegaGroup.json and  the change came in with this commit https://github.com/pravega/pravega/commit/06c6bc53e501f26712194788955919d4f37af5ff#diff-3ca244c82e953ba6417156aac99d3bc6R80.
Also,  that json deploys pravega in groups, so there will be a difference in the way of deployment.